src/cmd-buildextend-live

```diff
@@ -801,7 +862,21 @@ with open(build_semaphore, 'w') as f:
     f.write(f"{time.time_ns()}")
 
 try:
-    generate_iso()
+    if COSA_USE_LIVEISO is None:
```

Comment:
It will never be `None` because we give an empty string fallback. So I think we probably want to change this to

```suggestion
    if COSA_USE_LIVEISO != "":
```